other/materials_designer/Introduction.ipynb

```diff
@@ -25,6 +25,10 @@
     "\n",
```

Comment:
<p>Remove "bulk material" from 1.4 and its explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ok</p>